tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -696,8 +709,8 @@ function cleanup_containerd() {
 		return
 	fi
 
-	rm -f $containerd_conf_file
 	if [ -f "$containerd_conf_file_backup" ]; then
+ 		rm -f $containerd_conf_file
```

Comment:
What's the reason of this change here?

Reply:
If, for any reason, the backup has been deleted, you will lose the containerd configuration file.
With this change, you'll have a configuration file that requires manual cleanup but will still work.

But you're right, that's not directly related with microk8s support.

Reply:
I'd leave this one out, as I remember having some issues in the past in some cases when taking this approach.  I just don't remember which case :-/

Reply:
I've removed it. Should I also rebase on main ?

Reply:
If possible, yes, always a good approach. :-)